src/util.rs

```diff
@@ -175,6 +175,7 @@ impl std::task::Wake for NullWake {
 /// Block on a future, polling the device as needed.
 ///
 /// This will deadlock if the future is awaiting anything other than GPU progress.
+#[cfg(feature = "wgpu")]
```

Comment:
Why is this double `cfg(wgpu)`ed?

Reply:
My bad - I later realized this entire file was just wgpu helpers.